src/test/java/org/opensearch/flowframework/FlowFrameworkRestTestCase.java

```diff
@@ -706,4 +708,22 @@ protected GetPipelineResponse getPipelines() throws IOException {
             return GetPipelineResponse.fromXContent(parser);
         }
     }
+
+    @SuppressWarnings("unchecked")
+    protected List<String> catPlugins() throws IOException {
+        Response response = TestHelpers.makeRequest(
+            client(),
+            "GET",
+            "_cat/plugins?s=component&h=name,component,version,description&format=json",
+            null,
+            "",
+            ImmutableList.of(new BasicHeader(HttpHeaders.USER_AGENT, ""))
+        );
+        List<Object> pluginsList = JsonXContent.jsonXContent.createParser(
+            NamedXContentRegistry.EMPTY,
+            LoggingDeprecationHandler.INSTANCE,
+            response.getEntity().getContent()
+        ).list();
+        return pluginsList.stream().map(o -> ((Map<String, Object>) o).get("component").toString()).collect(Collectors.toList());
+    }
```

Comment:
We can use the same method [here](https://github.com/opensearch-project/flow-framework/blob/main/src/test/java/org/opensearch/flowframework/FlowFrameworkPluginIT.java#L23) as well